### DISCUSSION TITLE
Ao clicar para fazer o Fork, nada aparece!

### DISCUSSION THREAD

**Discussion by Rnetoo** [digitalinnovationone/dio-lab-open-source#49651] [Desafios DIO]
Quando peço para fazer o fork, a página recarrega e não encontro nada no meu perfil. 
Alguém ajuda por favor?

**Comment by kentonbell** [comment]
Olá! 👋

Quando você clica em **Fork**, o GitHub cria uma cópia do repositório na sua conta. Às vezes a página apenas recarrega e pode parecer que nada aconteceu.

Tente verificar:

1. Acesse o seu perfil no GitHub.
2. Clique na aba **Repositories**.
3. Procure pelo repositório **dio-lab-open-source**.

Você também pode acessar diretamente:

https://github.com/seu-usuario/dio-lab-open-source

Caso o fork realmente não tenha sido criado:
- Tente fazer o fork novamente.
- Verifique se não há bloqueadores de script no navegador.
- Tente usar outro navegador ou abrir em aba anônima.

Se o problema continuar, compartilhe um print da tela para que possamos entender melhor o que está acontecendo. 🙂
